client/wfprev-war/src/main/angular/src/app/services/map-config.service/layers/wildfires/being-held-wildfires.config.ts

```diff
@@ -0,0 +1,34 @@
+import { layerSettings } from '..';
+
+export function BeingHeldWildfiresLayerConfig(ls: layerSettings) {
+     return {
+      type: 'vector',
+      id: 'active-wildfires-holding',
+      title: 'Being Held Wildfires',
+      isQueryable: false,
```

Comment:
if we turn on identify, please change this isQueryable to true. Applys to on the layer config files

Reply:
done